crates/code_generation/src/write_to_file.rs

```diff
@@ -40,3 +40,20 @@ impl WriteToFile for TokenStream {
         Ok(())
     }
 }
+
+impl WriteToFile for String {
```

Comment:
This is basically just code duplication of impl for TokenStream. How can I reuse that function?